web/src/app/util/Chips.tsx

```diff
@@ -105,7 +106,33 @@ export function ScheduleChip(props: WithID<ChipProps>): JSX.Element {
 }
 
 export function SlackChip(props: WithID<ChipProps>): JSX.Element {
-  const { id: channelID } = props
+  const { id: channelID, ...rest } = props
+
+  const query = gql`
+    query($id: ID!) {
+      slackChannel(id: $id) {
+        teamID
```

Comment:
Include `id` in the requested fields so that Apollo's cache can differentiate, additionally we can set the policy to `cache-first`